### PR TITLE
US130912: LX > Investigate extremely long Build times

### PR DESCRIPTION
**Pull request by coffeexcode**
## Relevant Rally Links

- [US130912](https://rally1.rallydev.com/#/289692574792d/custom/355050439968?detail=%2Fuserstory%2F606827524297&fdp=true?fdp=true): LX > Investigate extremely long Build times

## Description
This PR is to reduce the time it takes for CI to run the test step. Based on the Brightspace support browsers found [here](https://documentation.brightspace.com/EN/brightspace/requirements/all/browser_support.htm) and the related native support for shadow DOM elements [here](https://caniuse.com/shadowdomv1) we should be able to safely reduce the numbers of tests for this repository by only testing the components using the browers native shadow DOM. 

Before, we would be testing the same component multiple times using the native shadow DOM, along with a polyfill, and again using `lazyRegister` which doesn't seem to be supported in Polymer 3. Some components we're also being tested a 4th time with no explicit properties (defaulting to native shadow DOM if available, otherwise polyfilled)

## Results
Based on the recent CI runs, and the latest CI run from this PR, we should see a reduction of CI times by up to 80% (>50% reduction for all runs)

Before:
image

After:
image

### CODE REVIEW

**Comment by ianlaidlaw** [comment]
look at those savings!!

File: test/testSuites.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+export const allTests = [
```
Comment (ianlaidlaw) [inline test/testSuites.js:1]: jw are these being used anywhere?

**Review by ianlaidlaw — APPROVED** [review]

File: test/testSuites.js (hunk 1 above)
Comment (coffeexcode) [reply test/testSuites.js:1]: Not really, I did think about removing it, but if someone wanted to test locally using the full suite then they'd have to pull in all the test cases from somewhere. CI now only uses the `nativeShadowDomTests`

Comment (ianlaidlaw) [reply test/testSuites.js:1]: fair enough, no harm keeping it like this in case people want to use them

**Pull request merged**